lib/src/support/theme.cpp

```diff
@@ -73,6 +73,9 @@ namespace cycfi { namespace elements
 
     , dialog_button_size         { 100 }
     , message_textbox_size       { { 300, 120 } }
+
+    , dial_mode                  { }
```

Comment:
I suggest making this explicit, instead of default construct. I also suggest making `linear` the default as it seems more common.